tests/test_opds.py

```diff
@@ -826,13 +826,13 @@ def test_groups_feed(self):
 
             feed = AcquisitionFeed.groups(
                 self._db, "test", self._url, fantasy_lane, annotator, 
-                False, False
+                force_refresh=False, use_materialized_works=False
             )
             eq_(CachedFeed.PAGE_TYPE, feed.type)
 
             cached_groups = AcquisitionFeed.groups(
                 self._db, "test", self._url, fantasy_lane, annotator, 
-                True, False
+                force_refresh=True, use_materialized_works=False
```

Comment:
FWIW, you can update the materialized views with `SessionManager.refresh_materialized_views(self._db)`. There are some examples in test_opds.py.